css/index.css

```diff
@@ -127,3 +127,93 @@ table {
   border-collapse: collapse;
   border-spacing: 0;
 }
+/* Coding Starts here */
+
+container{
+  width: 800px;
+  margin: auto;
+}
+ /* NO CSS for head!! */
+
+.body{
+  align-items: center;
+  max-width: 100px;
+  margin: 0 auto;
+  background-color: azure;
+}
+
+.nav{
+  flex-direction: row;
+  justify-content: flex-end;
+  font-size: 20px;
+
+  
```

Comment:
This .nav declaration isn't doing anything. See the comment below for more information as to why. 

Needs to be `nav` in this scenario.